test/lib/getNextJobs.test.js

```diff
@@ -20,15 +20,23 @@ describe('getNextJobs', () => {
         );
     });
 
-    it('should figure out what jobs start next', () => {
-        // trigger for a stage setup with the startFrom as a stage setup
+    it.only('should figure out what jobs start next', () => {
```

Comment:
This needs to be removed
```suggestion
    it('should figure out what jobs start next', () => {
```